bootstrap-app/src/cmr/bootstrap/api/routes.clj

```diff
@@ -30,45 +30,55 @@
   [params]
   (= "true" (when (:synchronous params) (str/lower-case (:synchronous params)))))
 
+(defn- get-dispatcher
```

Comment:
Agreed I moved it once but didn't like it here or in the bootstrap service namespace and figured I should just get a PR submitted and continue thinking of where to move it. That sounds like a good home.